<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DFTDialog</class>
 <widget class="QDialog" name="DFTDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DFT</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0">
   <item>
    <widget class="QCheckBox" name="removePadding">
     <property name="text">
      <string>Remove possible padding from last TDR</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="revertWindow">
     <property name="text">
      <string>Revert window function from last TDR</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>DC frequency (center)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="DCautomatic">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Automatic</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="DCmanual">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Specify manually:</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Frequency:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SIUnitEdit" name="freq">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="windowBox">
     <property name="title">
      <string>Window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
